yt_dlp/extractor/parliamentliveuk.py

```diff
@@ -11,33 +19,60 @@ class ParliamentLiveUKIE(InfoExtractor):
     _TESTS = [{
         'url': 'http://parliamentlive.tv/Event/Index/c1e9d44d-fd6c-4263-b50f-97ed26cc998b',
         'info_dict': {
-            'id': '1_af9nv9ym',
+            'id': 'c1e9d44d-fd6c-4263-b50f-97ed26cc998b',
             'ext': 'mp4',
             'title': 'Home Affairs Committee',
-            'uploader_id': 'FFMPEG-01',
-            'timestamp': 1422696664,
-            'upload_date': '20150131',
+            'timestamp': 1395153872,
+            'upload_date': '20140318',
+        },
+        'params': {
+            'format': 'bestvideo',
         },
     }, {
         'url': 'http://parliamentlive.tv/event/index/3f24936f-130f-40bf-9a5d-b3d6479da6a4',
         'only_matching': True,
     }]
+    _DEVICE_ID = str(uuid.uuid4())
+    _DATA = json.dumps({'device': {
+        'deviceId': _DEVICE_ID,
+        'width': 653,
+        'height': 368,
+        'type': 'WEB',
+        'name': ' Mozilla Firefox 91'},
+        'deviceId': _DEVICE_ID}).encode('utf-8')
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
-        webpage = self._download_webpage(
-            'http://vodplayer.parliamentlive.tv/?mid=' + video_id, video_id)
-        widget_config = self._parse_json(self._search_regex(
-            r'(?s)kWidgetConfig\s*=\s*({.+});',
-            webpage, 'kaltura widget config'), video_id)
-        kaltura_url = 'kaltura:%s:%s' % (
-            widget_config['wid'][1:], widget_config['entry_id'])
-        event_title = self._download_json(
-            'http://parliamentlive.tv/Event/GetShareVideo/' + video_id, video_id)['event']['title']
+        video_info = self._download_json(f'https://www.parliamentlive.tv/Event/GetShareVideo/{video_id}', video_id)
+
+        thumbnail = video_info.get('thumbnailUrl')
+        event_title = video_info['event']['title']
+        timestamp = unified_timestamp(try_get(video_info, lambda x: x['event']['publishedStartTime']))
+        auth = 'Bearer ' + self._download_json(
```

Comment:
move thumbnails, title and timestamp near the return or maybe even directly inside the return